h5netcdf/tests/test_h5netcdf.py

```diff
@@ -277,7 +279,10 @@ def read_legacy_netcdf(tmp_netcdf, read_module, write_module):
 
 
 def read_h5netcdf(tmp_netcdf, write_module):
-    remote_file = True if tmp_netcdf.startswith(remote_h5) else False
+    remote_file = False
+    if isinstance(tmp_netcdf, str):
+        if tmp_netcdf.startswith(remote_h5):
+            remote_file = True
```

Comment:
should be:
`remote_file = isinstance(tmp_netcdf, str) and tmp_netcdf.startswith(remote_h5)`

Reply:
thanks @shoyer, all great recommendations!